platform/Task.roc

```diff
@@ -217,3 +219,35 @@ batch = \current -> \next ->
         f <- next |> await
 
         map current f
+
+## Apply a task repeatedly to a list of items, and return a list of the resulting values
+##
+## ```
+## authors : List ID
+## getAuthor : ID -> Task Author [DbError]
+##
+## getAuthors : Task (List Author) [DbError]
+## getAuthors = Task.list authors getAuthor
+## ```
+##
+seq : List (Task ok err) -> Task (List ok) err
+seq = \tasks ->
+    List.walk tasks (Task.ok []) \state, task ->
+        value <- task |> Task.await
+
+        state |> Task.map \values -> List.append values value
```

Comment:
I think the comment doesn't fit the implementation. It seems to me, `Task.seq` actually converts a `List (Task ok err)` to a `Task (List ok) err`.

Or is there a `Task.list` function missing here?

Reply:
Thank you @RMailaender 

This implementation is copied from basic-cli. https://github.com/roc-lang/basic-cli/blob/307f1f91d984ac966cca75a40d403bdced1f05a6/platform/Task.roc#L227

I think if we are going to change this, we should discuss this further on zulip, and try to keep the API for basic-cli and basic-web server aligned.